openquake/qa_tests_data/event_based/case_8/expected/rup_data.csv

```diff
@@ -0,0 +1,3 @@
+rupserial,multiplicity,numsites,occurrence_rate,mag,lon,lat,depth,strike,dip,rake,boundary,ztor
+3,1,3,NAN,7.800000E+00,148.66792,43.11786,2.450000E+01,5.305669E+01,3.007192E+01,9.000000E+01,((148.14714 43.02214,148.19597 43.04970,148.24485 43.07725,148.29377 43.10477,148.34273 43.13227,148.39174 43.15976,148.44080 43.18722,148.48989 43.21466,148.53904 43.24207,148.58822 43.26947,148.63846 43.29587,148.68874 43.32224,148.73777 43.34985,148.78685 43.37743,148.83598 43.40499,148.88515 43.43253,148.93841 43.45575,148.97080 43.42471,149.00315 43.39366,149.03546 43.36259,149.06775 43.33153,149.10000 43.30045,149.13221 43.26936,149.16440 43.23826,149.19655 43.20716,149.14319 43.18394,149.09390 43.15640,149.04466 43.12884,148.99547 43.10126,148.94631 43.07366,148.89593 43.04728,148.84558 43.02089,148.79628 42.99349,148.74702 42.96607,148.69781 42.93863,148.64864 42.91117,148.59952 42.88369,148.55044 42.85619,148.50141 42.82867,148.45242 42.80112,148.40348 42.77356,148.37155 42.80466,148.33959 42.83576,148.30759 42.86684,148.27557 42.89792,148.24351 42.92899,148.21142 42.96005,148.17930 42.99110,148.14714 43.02214)),1.450000E+01
```

Comment:
Yes, it is correct, instead of an occurrence rate nonparametric sources have a PMF distribution (Probability Mass Function, a distribution of discrete probabilities with sum 1).